core/clear-color/renderer.zig

```diff
@@ -0,0 +1,39 @@
+const mach = @import("mach");
+const gpu = @import("gpu");
+
+pub const Renderer = @This();
+
+var queue: *gpu.Queue = undefined;
+
+pub fn RendererInit(core: *mach.Core) void {
```

Comment:
can you move this into `main.zig` and make `queue` an struct member (just like `core: mach.Core`)?
btw functions must be `camelCase`

Reply:
Hi @alichraghi, thanks for the feedback, I will move it to the main.zig.

And this file was made before I found the Zig style guide. I will also fix this